app/src/components/Graph.js

```diff
@@ -9,6 +9,7 @@ import PropTypes from "prop-types";
 import { formatDate, getMoment } from "utils/timeFormat";
 import { formatScalarValue } from "utils/formatting";
 import cn from "classnames/bind";
+import moment from "moment";
```

Comment:
I'm using a custom library instead of directly using moment.
I created a wrapper function that is imported on line 9 `getMoment` is the same as using `moment`